routers/posts.py

```
@@ -1,29 +1,278 @@
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail={
+                    "status": "error",
+                    "data": {
+                        "message": "본문은 비어있을 수 없습니다."
+                    }
+                }
+            )
+        post["content"] = data.content.strip()
+
+    #  업데이트 갱신
+    post["updated_at"] = datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)